src/main/java/sx/blah/discord/handle/obj/Verification.java

```diff
@@ -0,0 +1,43 @@
+package sx.blah.discord.handle.obj;
+
+/**
+ * Represents a verification level for a guild
+ */
+public enum Verification {
+	/**
+	 * Represents a verification level of None
+	 */
+	NONE,
+	/**
+	 * Represents a verification level of Low
+	 */
+	LOW,
+	/**
+	 * Represents a verification level of Medium
+	 */
+	MEDIUM,
+	/**
+	 * Represents a verification level of (╯°□°）╯︵ ┻━┻
+	 */
+	HIGH;
+
+	/**
+	 * Gets the Verification object for an integer level, or null if outside 0-3
+	 *
+	 * @param level The verification level as an int.
+	 * @return The Verification object for the given level.
+	 */
+	public static Verification getByLevel(int level) {
```

Comment:
I'd rather never have this return null, and instead throw the out of bounds exception instead. Tracking down a NPE is much more difficult than an index out of bounds exception